src/libproc/proc_pid.rs

```diff
@@ -281,7 +283,15 @@ pub fn pidpath(pid: i32) -> Result<String, String> {
 
 #[cfg(target_os = "linux")]
 pub fn pidpath(pid: i32) -> Result<String, String> {
-    unimplemented!()
+    let exe_path = CString::new(format!("/proc/{}/exe", pid)).unwrap();
+    let mut buf: Vec<u8> = Vec::with_capacity(PROC_PIDPATHINFO_MAXSIZE - 1);
```

Comment:
This constant is defined based on the MacOS code I found... while probably not a problem in practice, I wonder if there is a source of the same maximum in linux we could use?

Reply:
I found PATH_MAX which I've now switched it to use. Just pushing as we speak